web/src/topic_list.ts

```diff
@@ -397,4 +406,6 @@ export function initialize({
         assert(stream_id !== undefined);
         active_widgets.get(stream_id)?.build();
     });
+
+    stream_topic_history.set_update_topic_list_for_stream(update_widget_for_stream);
```

Comment:
We typically do this sort of thing via having `ui_init.ts` pass the function into the `initialize` for the target module.

Reply:
But see the other comments, I think you won't need to do this with the callback-passing approach suggested in other comments.